wallet/wallet.go

```diff
@@ -87,7 +87,7 @@ type Wallet struct {
 
 	// Handlers for stake system.
 	stakeSettingsLock  sync.Mutex
-	voteBits           stake.VoteBits
+	defaultVoteBits    stake.VoteBits
```

Comment:
`voteBits` or `defaultVoteBits`?
I've kept it as `defaultVoteBits` here to provide indication to code readers of the possibility of setting _other_ votebits; but in many usage sites such as `w.VoteBits()`, there is no `default` prefix.
Harmonize all to `voteBits` / `defaultVoteBits` or okay as is?

Reply:
i'm fine with this as is